grails-app/views/irb/show.gsp

```diff
@@ -66,13 +67,22 @@
         </g:if>
     </div>
 
-    <div id="documents">
+   <div id="documents">
         <g:render template="/common/documentsPanel" model="${[
+          controller        : issue.controller,
+          issue             : issue,
+          attachmentTypes   : attachmentTypes,
+          storageDocuments  : storageDocuments
+        ]}"/>
+    </div>
+
+    <div id="documentsNew">
+        <g:render template="/projectDocument/index" model="[
                 controller        : issue.controller,
                 issue             : issue,
-                attachmentTypes   : attachmentTypes,
-                storageDocuments  : storageDocuments
-        ]}"/>
+                attachmentTypes   : attachmentTypes
+                ]"/>
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/react-bootstrap-table/4.3.1/react-bootstrap-table-all.min.css"/>
```

Comment:
I've seen this in the other PR too. It doesn't make sense here - it should either be in the component or a global style ref.